examples/benchmarks/bubblebench/simple-tree/webpack.config.cjs

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const fluidRoute = require("@fluid-example/webpack-fluid-loader");
+const path = require("path");
+const { merge } = require("webpack-merge");
+const webpack = require("webpack"); //to access built-in plugins
+
+module.exports = (env) => {
+	const isProduction = env?.production;
+
+	return merge(
+		{
+			entry: {
+				main: "./src/index.ts",
+			},
+			resolve: {
+				extensionAlias: {
+					".js": [".ts", ".tsx", ".js", ".cjs", ".mjs"],
+				},
+				extensions: [".ts", ".tsx", ".js", ".cjs", ".mjs"],
+			},
+			module: {
+				rules: [
+					{
+						test: /\.tsx?$/,
+						loader: "ts-loader",
+					},
+					{
+						test: /\.m?js$/,
+						use: [require.resolve("source-map-loader")],
+						enforce: "pre",
+					},
+				],
+			},
+			output: {
+				filename: "[name].bundle.js",
+				path: path.resolve(__dirname, "dist"),
+				library: "[name]",
+				// https://github.com/webpack/webpack/issues/5767
+				// https://github.com/webpack/webpack/issues/7939
+				devtoolNamespace: "fluid-example/bubblebench",
+				libraryTarget: "umd",
+			},
+			// This impacts which files are watched by the dev server (and likely by webpack if watch is true).
+			// This should be configurable under devServer.static.watch
+			// (see https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md) but that does not seem to work.
+			// The CLI options for disabling watching don't seem to work either, so this may be a symptom of using webpack4 with the newer webpack-cli and webpack-dev-server.
+			watchOptions: {
+				ignored: "**/node_modules/**",
+			},
+		},
+		isProduction ? require("./webpack.prod.cjs") : require("./webpack.dev.cjs"),
```

Comment:
personally, I don't find the pattern of having three webpack config files very useful. Just inlining the prod and dev configs in here seems like a much cleaner setup, and avoid the possible confusion of someone thinking they should use the prod or dev one directly based on what they are doing.

Reply:
I have a PR to do this change to another package: https://github.com/microsoft/FluidFramework/pull/20209 . Assuming people like that, we can ensure this package's webpack config reflects those improvements.

Reply:
Looks good, I could update the webpack accordingly, or alternatively waiting the task of updating them uniformly across the whole repo.